shapely/tests/test_strtree.py

```diff
@@ -1436,248 +1362,281 @@ def test_query_bulk_dwithin_invalid_distance(tree, distance, match):
+        (MultiPoint([[5, 7], [7, 5]]), 1.5, [5, 6]),
+        ([MultiPoint([[5, 7], [7, 5]])], 1.5, [[0, 0], [5, 6]]),
     ],
 )
-def test_query_bulk_dwithin_lines(line_tree, geometry, distance, expected):
+def test_query_dwithin_lines(line_tree, geometry, distance, expected):
     assert_array_equal(
-        line_tree.query_bulk(geometry, predicate="dwithin", distance=distance), expected
+        line_tree.query(geometry, predicate="dwithin", distance=distance),
+        expected,
     )
 
 
-@pytest.mark.skipif(shapely.geos_version < (3, 10, 0), reason="GEOS < 3.10")
+@pytest.mark.skipif(geos_version < (3, 10, 0), reason="GEOS < 3.10")
 @pytest.mark.parametrize(
     "geometry,distance,expected",
     [
-        (shapely.points(0, 0), 0, [[0], [0]]),
-        (shapely.points(0, 0), 0.5, [[0], [0]]),
-        (shapely.points(0, 0), 1.5, [[0, 0], [0, 1]]),
-        (shapely.points(0.5, 0.5), 1, [[0, 0], [0, 1]]),
-        (shapely.points(0.5, 0.5), 0.5, [[0, 0], [0, 1]]),
-        (box(0, 0, 1, 1), 0, [[0, 0], [0, 1]]),
-        (box(0, 0, 1, 1), 2, [[0, 0, 0], [0, 1, 2]]),
-        (shapely.multipoints([[5, 5], [7, 7]]), 0.5, [[0, 0], [5, 7]]),
+        (Point(0, 0), 0, [0]),
+        ([Point(0, 0)], 0, [[0], [0]]),
+        (Point(0, 0), 0.5, [0]),
+        ([Point(0, 0)], 0.5, [[0], [0]]),
+        (Point(0, 0), 1.5, [0, 1]),
+        ([Point(0, 0)], 1.5, [[0, 0], [0, 1]]),
+        (Point(0.5, 0.5), 1, [0, 1]),
+        ([Point(0.5, 0.5)], 1, [[0, 0], [0, 1]]),
+        (Point(0.5, 0.5), 0.5, [0, 1]),
+        ([Point(0.5, 0.5)], 0.5, [[0, 0], [0, 1]]),
+        (box(0, 0, 1, 1), 0, [0, 1]),
+        ([box(0, 0, 1, 1)], 0, [[0, 0], [0, 1]]),
+        (box(0, 0, 1, 1), 2, [0, 1, 2]),
+        ([box(0, 0, 1, 1)], 2, [[0, 0, 0], [0, 1, 2]]),
+        (MultiPoint([[5, 5], [7, 7]]), 0.5, [5, 7]),
+        ([MultiPoint([[5, 5], [7, 7]])], 0.5, [[0, 0], [5, 7]]),
         (
-            shapely.multipoints([[5, 5], [7, 7]]),
+            MultiPoint([[5, 5], [7, 7]]),
+            2.5,
+            [3, 4, 5, 6, 7, 8, 9],
+        ),
+        (
+            [MultiPoint([[5, 5], [7, 7]])],
             2.5,
             [[0, 0, 0, 0, 0, 0, 0], [3, 4, 5, 6, 7, 8, 9]],
         ),
     ],
 )
-def test_query_bulk_dwithin_polygons(poly_tree, geometry, distance, expected):
+def test_query_dwithin_polygons(poly_tree, geometry, distance, expected):
     assert_array_equal(
-        poly_tree.query_bulk(geometry, predicate="dwithin", distance=distance), expected
+        poly_tree.query(geometry, predicate="dwithin", distance=distance),
+        expected,
     )
 
 
 ### STRtree nearest
 
 
-@pytest.mark.skipif(shapely.geos_version < (3, 6, 0), reason="GEOS < 3.6")
+@pytest.mark.skipif(geos_version < (3, 6, 0), reason="GEOS < 3.6")
 def test_nearest_empty_tree():
-    tree = shapely.STRtree([])
+    tree = STRtree([])
     # TODO what do we want here?
-    assert tree.nearest_item(point) is None
+    assert tree.nearest(point) is None
```

Comment:
No substantive logic changes for `nearest` functions other than `nearest_item` => `nearest` and cleanup of geometry inputs / shapely namespace